beacon-chain/core/blocks/withdrawals.go

```diff
@@ -120,35 +120,36 @@ func ValidateBLSToExecutionChange(st state.ReadOnlyBeaconState, signed *ethpb.Si
 //
 // Spec pseudocode definition:
 //
-//	def process_withdrawals(state: BeaconState, payload: ExecutionPayload) -> None:
-//	    expected_withdrawals, partial_withdrawals_count = get_expected_withdrawals(state)  # [Modified in Electra:EIP7251]
+//		def process_withdrawals(state: BeaconState, payload: ExecutionPayload) -> None:
+//		    # [Modified in Electra:EIP7251]
+//	   	expected_withdrawals, processed_partial_withdrawals_count = get_expected_withdrawals(state)
```

Comment:
Can we revert these changes? I think the original format is fine?
Reason being it's hard to view the differences and the next person will just change it back agian